code/src/App.js

```diff
@@ -1,9 +1,76 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import ThoughtForm from './components/ThoughtForm'
+import ThoughtItem from './components/ThoughtItem'
+import LoadingItem from './components/Loading'
+
```

Comment:
I think it is convention for the component to have the same file name and function name (so the file number should be LoadingItem.js